rush.json

```diff
@@ -91,12 +91,6 @@
       "reviewCategory": "production",
       "shouldPublish": true
     },
-    {
```

Comment:
this probably should have been a separate change that went in before the OIDC change.

Reply:
Also, if this package is going away, does this need to go to? https://github.com/aws-solutions/solution-spark-on-aws/blob/49f18e511a91f97bb0498650794123d8d424dded/.github/workflows/build-and-test.yml#L30